README.md

```diff
@@ -129,6 +129,43 @@ If you intend to run repository tests:
 pip install -r requirements.txt
 ```
 
+
+<details><summary><strong><em>On Windows: </strong></em></summary>
```

Comment:
this is good for now, but I prefer the style in the lightgbm documentation where a whole section is devoted just to one OS as opposed to this interleaved style

Reply:
(it's much easier to follow)

Reply:
The problem is that most of it is the same, it's just these small sections

Reply:
where's pip install interpret-community?